app/components/buttons/ToggleSwitch.js

```diff
@@ -0,0 +1,27 @@
+import "style/ToggleSwitch.less";
```

Comment:
All new or refactored components should have `.jsx` extension to make it clear which includes plain `.js` and which `.jsx`